components/TARGET_PSA/services/inc/autogen_sid.h

```diff
@@ -0,0 +1,110 @@
+/* Copyright (c) 2019 ARM Limited
```

Comment:
Is this used by TFM? If not, consider putting into `components/TARGET_PSA/TARGET_MBED_SPM/services/inc/`.

Reply:
This is used even when TF-M SPM is used, so OK.